feature/interface/aggregate/otg_tests/aggregate_all_not_viable_test/aggregate_all_not_forwarding_viable_test.go

```diff
@@ -374,8 +374,12 @@ func configureDUT(t *testing.T, dut *ondatra.DUTDevice) []string {
 			}
 			if port.PMD() == ondatra.PMD100GBASEFR {
 				e.AutoNegotiate = ygot.Bool(false)
-				e.DuplexMode = oc.Ethernet_DuplexMode_FULL
-				e.PortSpeed = oc.IfEthernet_ETHERNET_SPEED_SPEED_100GB
+				if deviations.ExplicitDuplexModeRequired(dut) {
+					e.DuplexMode = oc.Ethernet_DuplexMode_FULL
+				}
+				if deviations.ExplicitPortSpeed(dut) {
+					e.PortSpeed = oc.IfEthernet_ETHERNET_SPEED_SPEED_100GB
+				}
```

Comment:
the whole block should be under a new deviation if there's a vendor who needs it